eth2/beacon/epoch_processing_helpers.py

```diff
@@ -138,75 +134,92 @@ def get_winning_root(
 
 
 @to_tuple
-@to_set
-def get_epoch_boundary_attester_indices(
+def get_attesting_indices(state: 'BeaconState',
+                          attestations: Sequence[PendingAttestationRecord],
+                          config: CommitteeConfig) -> Sequence[ValidatorIndex]:
+    output: Set[ValidatorIndex] = set()
+    for a in attestations:
+        participants = get_attestation_participants(state, a.data, a.aggregation_bitfield, config)
+        output = output.union(participants)
+    return sorted(list(output))
```

Comment:
Looks like it can sort the set directly
```suggestion
    return sorted(list(output))
    return sorted(output)
```

Reply:
nice! let's get that unnecessary list constructor out of there...